src/snowflake/snowpark/modin/plugin/docstrings/base.py

```diff
@@ -1832,6 +1832,15 @@ def loc():
         viper               0       0
         sidewinder          0       0
 
+        Setting the values with a Series item.
```

Comment:
@sfc-gh-helmeleegy this is the example I added

Reply:
Thanks!